apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -104,7 +105,10 @@ class ProgressBubble extends React.Component {
 
     let href = '';
     if (!disabled && url) {
-      href = url + location.search;
+      href = url;
+      if (selectedSectionId) {
+        href += `?section_id=${selectedSectionId}`;
```

Comment:
Will this work correctly if we have hashes in our URL (i.e. once we have progress bubbles in teacher-dashboard)?
 Could try by going to progress dashboard with `sectionProgressRedesign` experiment enabled.

Reply:
Great point. Is there a spec of some kind showing where these would get added to teacher dashboard?

As a means of future-proofing, I've modified the URL-checker in the UI test to make sure the query param comes before any `#`, so hopefully we'll catch this kind of problem if we ever point the UI test at teacher dashboard. I'm going to take Brad's suggestion and merge without addressing this for now. 

Brad, we should discuss whether this should be fixed soonish, or closer to when we start having progress bubbles in teacher dashboard.